doc/classes/Basis.xml

```diff
@@ -340,94 +370,124 @@
 	</methods>
 	<members>
 		<member name="x" type="Vector3" setter="" getter="" default="Vector3(1, 0, 0)">
-			The basis matrix's X vector (column 0). Equivalent to array index [code]0[/code].
+			The basis' X axis, and the column [code]0[/code] of the matrix.
```

Comment:
This is not correct grammar, `s'` is used if the word is both plural and has ownership, but `basis` is not plural.